changelog/fragments/modify-default-channel.yaml

```diff
@@ -0,0 +1,17 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      In pkgman-to-bundle command, modified the `channels` for manifests (other than latest version)
+      to be the `defaultChannel` from `package.yaml` instead of `candidate`.
```

Comment:
```suggestion
      In the `pkgman-to-bundle` command, changed the default channel name used for CSV's not
      specified in `package.yaml` to `defaultChannel` instead of "candidate".
```